[0m[[0mdebug[0m] [0mjavaOptions: List()[0m
[0m[[0minfo[0m] [0mRunning util.PstFinder [0m
[0m[[31merror[0m] [0mException in thread "main" java.io.EOFException[0m
[0m[[31merror[0m] [0m	at java.io.ObjectInputStream$PeekInputStream.readFully(ObjectInputStream.java:2335)[0m
[0m[[31merror[0m] [0m	at java.io.ObjectInputStream$BlockDataInputStream.readShort(ObjectInputStream.java:2804)[0m
[0m[[31merror[0m] [0m	at java.io.ObjectInputStream.readStreamHeader(ObjectInputStream.java:802)[0m
[0m[[31merror[0m] [0m	at java.io.ObjectInputStream.<init>(ObjectInputStream.java:299)[0m
[0m[[31merror[0m] [0m	at util.Data$.objectFromFile(Data.scala:22)[0m
[0m[[31merror[0m] [0m	at util.PstFinder$.main(PstFinder.scala:24)[0m
[0m[[31merror[0m] [0m	at util.PstFinder.main(PstFinder.scala)[0m
java.lang.RuntimeException: Nonzero exit code returned from runner: 1
	at scala.sys.package$.error(package.scala:27)
	at sbt.BuildCommon$$anonfun$toError$1.apply(Defaults.scala:2077)
	at sbt.BuildCommon$$anonfun$toError$1.apply(Defaults.scala:2077)
	at scala.Option.foreach(Option.scala:236)
	at sbt.BuildCommon$class.toError(Defaults.scala:2077)
	at sbt.Defaults$.toError(Defaults.scala:39)
	at sbt.Defaults$$anonfun$runTask$1$$anonfun$apply$38$$anonfun$apply$39.apply(Defaults.scala:750)
	at sbt.Defaults$$anonfun$runTask$1$$anonfun$apply$38$$anonfun$apply$39.apply(Defaults.scala:748)
	at scala.Function1$$anonfun$compose$1.apply(Function1.scala:47)
	at sbt.$tilde$greater$$anonfun$$u2219$1.apply(TypeFunctions.scala:40)
	at sbt.std.Transform$$anon$4.work(System.scala:63)
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:228)
	at sbt.Execute$$anonfun$submit$1$$anonfun$apply$1.apply(Execute.scala:228)
	at sbt.ErrorHandling$.wideConvert(ErrorHandling.scala:17)
	at sbt.Execute.work(Execute.scala:237)
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:228)
	at sbt.Execute$$anonfun$submit$1.apply(Execute.scala:228)
	at sbt.ConcurrentRestrictions$$anon$4$$anonfun$1.apply(ConcurrentRestrictions.scala:159)
	at sbt.CompletionService$$anon$2.call(CompletionService.scala:28)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at java.lang.Thread.run(Thread.java:745)
[0m[[31merror[0m] [0m(compile:[31mrun[0m) Nonzero exit code returned from runner: 1[0m
